### ISSUE TITLE
Not working on Microsoft Edge

### ISSUE THREAD

**Issue by FutoRicky** [rubo77/table2CSV#17]
It is currently working on all browsers I have tried (Chrome, Firefox & Safari) except Edge. I tried debugging it, but no console logs are shown. Any idea on this issue? Thanks!

**Comment by rubo77** [comment]
no, but maybe the other fork: https://github.com/ZachWick/TableCSVExport that already implemented some more new features

**Comment by FutoRicky** [comment]
@rubo77 Thanks for the response. After some investigating I found that Edge does not support the use of data URI in links which is the way this library downloads the csv. [StackOverflow reference](https://stackoverflow.com/a/33197664/3746396)

What I'm thinking about doing in my project is continue using this library but merge it with [download.js](https://github.com/rndme/download). I'll probably make a fork just in case anyone is facing the same issue and wants a solution.
